lib/xlsx_writer/xml/workbook_rels.erb

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <Relationships xmlns="http://schemas.openxmlformats.org/package/2006/relationships">
-  <Relationship Id="rId0" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/sharedStrings" Target="sharedstrings.xml"/>
```

Comment:
@seamusabshere re: #12 this is the only reference to "sharedstrings.xml" (all lower-case) that remains.